.github/dependabot.yml

```diff
@@ -12,6 +12,19 @@ updates:
         patterns:
           - "*"
   - package-ecosystem: github-actions
+    target-branch: master
+    directory: /
+    schedule:
+      interval: weekly
```

Comment:
Just FYI we might want to change these to two weeks since it seems to take a while to get them merged